dockers/docker-fpm-frr/frr/supervisord/bgp_eoiu_marker.conf

```diff
@@ -0,0 +1,11 @@
+[program:bgp_eoiu_marker]
```

Comment:
What is the reason you broke bgp_eoiu_marker and vtysh_b out into their own conf files as opposed to simply including these sections in the main supervisor.conf file?

Reply:
you no longer use supervisor ctl to control start.  once it is part of supervisor conf they will be started by the dependent start script

Reply:
My question was: why did you decide to create separate conf files for these two processes rather than adding them into the main dockers/docker-fpm-frr/supervisord.conf file? I think we should be consistent: either all processes in their own files or all in one.

Reply:
move all into j2 template